server/routes/index.js

```diff
@@ -29,4 +29,13 @@ router.post('/charge', (req, res) => {
   });
 });
 
+router.post('/addevent', (req, res) => {
+  const databaseInsertSuccess = true;
+  if (databaseInsertSuccess) {
+    res.status(200).json({ addSuccess: true, message: 'YAY' });
+  } else {
+    res.status(200).json({ addSuccess: false, message: 'WOMP WOMP' });
```

Comment:
@GameKyuubi we are going to change error status from 200 to something else later, aren't we?

Reply:
fixed